.pre-commit-hooks.yaml

```
@@ -108,3 +108,5 @@
   description: Runs terrascan on Terraform templates.
   language: script
   entry: terrascan.sh
+  files: \.tf$
+  exclude: \.+.terraform\/.*$
```

@MaxymVlasov Should this be this?
```suggestion
  exclude: \.terraform\/.*$
```